core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/BufferStateForwarder.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.PlaybackParameters;
+import com.google.android.exoplayer2.Timeline;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+import com.novoda.noplayer.listeners.BufferStateListeners;
+
+class BufferStateForwarder implements ExoPlayer.EventListener {
+
+    private final BufferStateListeners bufferStateListeners;
+
+    BufferStateForwarder(BufferStateListeners bufferStateListeners) {
+        this.bufferStateListeners = bufferStateListeners;
+    }
+
+    @Override
+    public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
+        if (playbackState == ExoPlayer.STATE_BUFFERING) {
+            bufferStateListeners.onBufferStarted();
+        } else if (playbackState == ExoPlayer.STATE_READY) {
+            bufferStateListeners.onBufferCompleted();
+        }
+    }
+
+    @Override
+    public void onTimelineChanged(Timeline timeline, Object manifest) {
+        //TODO should we send ?
```

Comment:
Same here, we need to figure out how we should map these from `ExoPlayer` to `no-player`